shared_model/validators/settings.hpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_SHARED_MODEL_SETTINGS_HPP
+#define IROHA_SHARED_MODEL_SETTINGS_HPP
+
+#include "interfaces/common_objects/types.hpp"
+
+namespace shared_model {
+
+  namespace validation {
+
+    /**
+     * Structure that holds configurable ledger settings
+     */
+    struct Settings {
+      explicit Settings(size_t max_description_size = 64);
```

Comment:
this structure does not need a constructor

Reply:
For tests
```C++
static const std::shared_ptr<const shared_model::validation::Settings>
        kTestSettings =
            std::make_shared<const shared_model::validation::Settings>(128);
```

Reply:
I still think that a constructor is not needed. If this structure ever gets big, it will be hard to track which value initializes which member (imagine counting argument numbers in constructor call and then comparing it to constructor definition). Big plain structures are easier to initialize through member access IMHO. Same for tests.